tests/test-faceting.php

```diff
@@ -214,4 +214,50 @@ function test_author_facets() {
 		$this->assertEquals( 'author_b', $facet_data['Author']['items'][1]['name'] );
 		$this->assertEquals( 1, $facet_data['Author']['items'][1]['count'] );
 	}
+
+	function test_facet_by_taxonomy() {
+		// Fake a taxonomy query to WP_Query so the query vars are set properly.
+		global $wp_query;
+		$wp_query = new WP_Query(
```

Comment:
nitpicky, but this will trigger a database call... maybe there's no way around that, but if there is, it would be nice to avoid.

Reply:
I think that might be possible - I'll give it a try.

Reply:
Yep, turns out you can just `$wp_query->parse_query( $new_query )` and it works like a charm. I added that and a few other cleanup items that I noticed while setting this up on my new m1 (phpunit polyfills path, trusting phpcs in composer.json, a few other places to use `capital_p_dangit`)